zebra-chain/src/block/header.rs

```diff
@@ -35,10 +34,19 @@ pub struct BlockHeader {
     /// header.
     pub merkle_root_hash: MerkleTreeRootHash,
 
-    /// [Sapling onward] The root LEBS2OSP256(rt) of the Sapling note
+    /// [Pre-Sapling] Reserved. All zeroes.
+    /// [Sapling and Blossom] The root LEBS2OSP256(rt) of the Sapling note
     /// commitment tree corresponding to the final Sapling treestate of
     /// this block.
-    pub final_sapling_root_hash: SaplingNoteTreeRootHash,
+    /// [Heartwood onward] The root of a Merkle Mountain Range tree, which
+    /// commits to various features of the chain's history, including the
+    /// Sapling commitment tree. This commitment supports the FlyClient
+    /// protocol. See ZIP-221 for details.
+    // TODO:
+    //   - replace with an unspecified HistoryRootHash type?
+    // Note that the NetworkUpgrade list is in zebra-consensus, so we can't
+    // parse this field into a HistoryRootHash enum in zebra-chain.
+    pub history_root_hash: [u8; 32],
```

Comment:
As suggested by @daira based on the spec:
```suggestion
    pub light_client_root_hash: [u8; 32],
```